src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -221,7 +277,28 @@ export default {
 				this.$kvTrackEvent('Register', 'error-register-social-form-invalid-input');
 			}
 		},
+		fetchStrategicPartnerLoginInfoByPageId() {
```

Comment:
Should be able to remove this and the call in mounted now...

Reply:
Ack.  I thought it was the prefetch that was successfully fetching the logo but it looks like it is still this function.  When I delete it, it seems like the graphql call doesn't even get made.  I'm not sure why.

Reply:
YOu need a "result(result) {}" block after your prefetch function where you can set values on `this`. `this` cannot be set in the direct response/callback of the prefetch query.

Reply:
Does this look right to you?  I get a 500 server error when I do this.
![image](https://github.com/kiva/ui/assets/3431240/59b09787-13f1-4f7b-924b-47ee085e03a5)

Reply:
![image](https://github.com/kiva/ui/assets/3431240/a10a0466-636a-4e68-96d3-752e9f299a92)
This too

Reply:
This also doesn't work
![image](https://github.com/kiva/ui/assets/3431240/2e66b3ed-f39c-43e5-af01-ee3df0e8af81)

Reply:
oh, it looks like this.partnerContentId can't be accessed.

Reply:
Oh.. also, looks like you need a prefechVariables block... here's a good example: https://github.com/kiva/ui/blob/main/src/pages/BorrowerProfile/BorrowerProfile.vue#L517

Reply:
you'll prob want the `variables` block too. prefetchVariables are for the server fetch before the vue functionality is initialized and variables are in the clicet for an in-app navigation...

Reply:
500 here too 
![image](https://github.com/kiva/ui/assets/3431240/89a3f12b-5637-4f4e-a0ba-e47c2dd22b37)

Reply:
pushed this latest non-working version just now.

Reply:
Can't use `this.` in the variables. you'll need to pass in the `{ route }` object to both variable blocks and extract the partnerContentId directly from the route.

Reply:
okay so this makes the graphql fetch call, but variables are not set for the query.

![image](https://github.com/kiva/ui/assets/3431240/600a63c7-d994-47c0-ace6-66e19168fe61)


Then this following version results in an error that says [Vue warn]: Error in created hook: "TypeError: Cannot read properties of undefined (reading 'route')"

![image](https://github.com/kiva/ui/assets/3431240/c0db9425-b69a-424c-bfcf-1d9b9850ba26)

And then this appears to make no graphql call at all.
![image](https://github.com/kiva/ui/assets/3431240/65722c71-b1ce-42ad-890f-0563a4a35f84)

Reply:
In LoanStory.vue I see an implementation here that uses route in the preFetchVariables and uses this for variables.
![image](https://github.com/kiva/ui/assets/3431240/da8016b0-c7a2-4f47-bc3f-79f58db37f86)

So it seems like that's possible to do, but not sure then why the variable isn't getting set in the first of the three.